apps/src/aichat/views/modelCustomization/RetrievalCustomization.tsx

```diff
@@ -110,13 +103,7 @@ const RetrievalCustomization: React.FunctionComponent = () => {
         })}
       </div>
       <div className={modelCustomizationStyles.footerButtonContainer}>
-        <Button
-          text="Update"
-          onClick={onUpdate}
-          iconLeft={{iconName: 'edit'}}
-          className={modelCustomizationStyles.updateButton}
-          disabled={isReadOnly}
-        />
+        <UpdateButton allFieldsDisabled={isReadOnly} />
```

Comment:
I don't think this prop name makes sense in this context...`isDisabled` isn't quite right either since the button can also be disabled based on `saveInProgress` within this button, but seems closer at least (`disabledByDefault`?)

Reply:
I think this is okay in that for the button,  `disabled={allFieldsDisabled || saveInProgress}`.

Reply:
Sorry maybe I wasn't clear, I'm saying the prop name `allFieldsDisabled` doesn't make sense to me here, since we're passing in the value `isReadOnly`?

I'm guessing Sanchit started refactoring from `SetupCustomization` (where this prop name makes sense), then re-used it here without picking a more general prop name?

Reply:
Gotcha.  `isDisabled` does seem a bit better.

Reply:
Nice rename.